scripts/inference

```diff
@@ -18,4 +18,11 @@ then
 fi
 
 distDir=$myDir"/../dist"
-eval "java -classpath "$distDir"/checker.jar:"$distDir"/plume.jar:"$distDir"/checker-framework-inference.jar  checkers.inference.InferenceLauncher " "$@"
+classpath="$distDir"/checker.jar:"$distDir"/plume.jar:"$distDir"/checker-framework-inference.jar
+
+if [ "$thrid_party_checker_classpath" != "" ] ; then
```

Comment:
The downstream project will set this env var, For example, GTIS can set this env var in its [gralde file](
https://github.com/opprop/generic-type-inference-solver/blob/a6b946f815ca17e2461ec8ebea94ae77610505a1/build.gradle#L132)